### PR TITLE
Slice add == and bytesize methods.

### PR DESCRIPTION
**Pull request by technorama**

### CODE REVIEW

File: src/slice.cr [hunk 1]
```diff
@@ -240,6 +240,20 @@ struct Slice(T)
     ((c < 10 ? 48_u8 : 87_u8) + c)
   end
 
+  def to_unsafe
+    @pointer
+  end
```
Comment (jhass) [inline src/slice.cr]: `to_unsafe` is already there though ;)

**Comment by technorama** [comment]
@jhass Must have mistyped when searching for it.  Fixed.

**Comment by jhass** [comment]
LGTM, thanks!

**Comment by asterite** [comment]
I wonder if slices should really be compared by content, like array and static array, invoking == on each pair

**Comment by jhass** [comment]
I see arguments for (symmetry with the rest of stdlib) and against (slow) it. Given Slice is low level primitive, I think we can try this variant for a while and see if it's harmful in practice.

**Comment by technorama** [comment]
A more comprehensive PR is coming with correct comparisons.

**Comment by Nakilon** [comment]
Note 4 usual 'equality check' operators in Ruby: http://stackoverflow.com/a/7157051/322020
And that you may also check the pointer addresses for faster return.

**Pull request merged**